tools/dashboard-plugin-tests/plugins/plugin-8.8.0/.gitignore

```diff
@@ -1,3 +1,2 @@
 .env.secrets
 dist
-node_modules
```

Comment:
Is this intentional? Or do the node_modules get ignored by some other gitignore?

Reply:
This is interesting, it's basically generated from the plugin cli tool, previous plugin was just removed... But it's likely not necessary, as we have `node_modules/` in gitignore in the root folder of the SDK